GitUI/ScriptsEngine/ScriptsManager.ScriptRunner.cs

```diff
@@ -16,28 +16,27 @@ internal static class ScriptRunner
             private const string PluginPrefix = "plugin:";
             private const string NavigateToPrefix = "navigateTo:";
 
-            public static CommandStatus RunScript<THostForm>(ScriptInfo script, THostForm form, RevisionGridControl? revisionGrid)
-                where THostForm : IGitModuleForm, IWin32Window
+            public static bool RunScript(ScriptInfo script, IScriptHostControl scriptHostControl)
             {
                 try
                 {
-                    return RunScriptInternal(script, form, form.UICommands, revisionGrid);
+                    return RunScriptInternal(script, scriptHostControl.Window, scriptHostControl.UICommands, scriptHostControl);
```

Comment:
```suggestion
                    return RunScriptInternal(script, scriptHostControl);
```

Reply:
I'm fairly sure the split is intentional. This is the only place where `RunScriptInternal` is called, so it exists purely to perform parameter unpacking, and to allow catching an exception.

The only other use is in tests, where being able to pass `null` in `UICommands` is necessary in order to easily trigger an error. Changing this could invalidate some of the tests.